code/game/turfs/simulated/walls_reinforced.dm

```diff
@@ -9,23 +9,49 @@
 
 	var/d_state = 0
 
+/turf/simulated/wall/r_wall/examine(mob/user)
+	..()
+	if(d_state)
+		switch(d_state) //How fucked or unfinished is our wall
+			if(1)
+				desc += "It has no outer grille"

```

Comment:
This isn't going to work.
Not only are you going to extend the description each time someone examines it, this is called after the code that shows the mob the desc anyways. Make these user << or user.show_message()